src/components/manager/projects/ProjectJsonSchema.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { projectJsonSchemaTypesShape } from "../../../propTypes";
+import { Button, Card, Modal, Typography, Tooltip } from "antd";
+import ReactJson from "react-json-view";
+import PropTypes from "prop-types";
+import { useDispatch, useSelector } from "react-redux";
+import { deleteProjectJsonSchema } from "../../../modules/metadata/actions";
+
+const ProjectJsonSchema = ({ projectSchema }) => {
+    const dispatch = useDispatch();
+    const handleDelete = () => {
+        const deleteModal = Modal.confirm({
+            title: `Are you sure you want to delete the "${projectSchema.schema_type}" project JSON schema?`,
+            content: <>
+                <Typography.Paragraph>
+                    Doing so will mean that extra_properties data validation will not be
+                    enforced for entities of type <strong>{projectSchema.schema_type}</strong>
+                    in project {projectSchema.project}
+                </Typography.Paragraph>
+            </>,
+            width: 720,
+            autoFocusButton: "cancel",
+            okText: "Delete",
+            okType: "danger",
+            maskClosable: true,
+            onOk: async () => {
+                deleteModal.update({ okButtonProps: { loading: true } });
+                await dispatch(deleteProjectJsonSchema(projectSchema));
+                deleteModal.update({ okButtonProps: { loading: false } });
+            }
+        })
+    }
+    return (
+        <Card
+            key={projectSchema.id}
+            title={projectSchema.schema_type}
+            extra={
+                <Button type="danger" icon="delete" onClick={handleDelete}>Delete</Button>
+            }
+        >
+            <Typography.Paragraph>
```

Comment:
or maybe an uncontrolled form with no settable fields

Reply:
Forgot this one, just changed it to descriptions. Went for vertical layout with one column, otherwise expanding and collapsing the json makes the columns width vary.